Makefile.am

```diff
@@ -5,14 +5,14 @@ SUBDIRS = lib src man
 EXTRA_DIST = m4/gnulib-cache.m4
 
 install-exec-local:
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfcat
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfcp
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfls
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfmkdir
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfmv
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfrm
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfstat
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gftail
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfcat
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfcp
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfls
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfmkdir
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfmv
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfrm
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfstat
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gftail
```

Comment:
@craigcabrey , I don't know if I understood your reply correctly. With the current code base, links were not available under standard binary paths. They were created directly under root of the source. During packaging we overcome this behaviour by re-creating links to gfcli in for loop via spec file. I thought of doing it as part of 'make install' and hence made the above change.

@nixpanic , if we create relative links like the one you mentioned above, that would mean those are links to the path where gfcli binary is present inside source which is not our intention. In the absence of source those links will be dead, right? Please correct me if I am wrong.